documentation-src/metalsmith/content/reference.md

```diff
@@ -13,6 +13,8 @@ during the search cycle with events:
  - `result`: when the results are retrieved from Algolia
  - `error`: when Algolia sends back an error
  - `searchQueueEmpty`: when there is no more on going searches
```

Comment:
ongoing means that there are searches that are busy, while pending means that there's something to happen with them still. Both almost the same